src/main/java/life/qbic/subscriptions/encryption/RequestEncrypter.java

```diff
@@ -0,0 +1,9 @@
+package life.qbic.subscriptions.encryption;
+
+import life.qbic.subscriptions.subscriptions.CancellationRequest;
+
+public interface RequestEncrypter {
+
+  String encryptCancellationRequest(CancellationRequest cancellationRequest)
+      throws EncryptionException;
```

Comment:
```suggestion
  String encryptCancellationRequest(CancellationRequest cancellationRequest) throws EncryptionException;
```

Reply:
I do not understand this suggestion? The indentation is intentional to highlight the continuation of the line and adheres to the google style we use.